LESS/about.less

```diff
@@ -0,0 +1,36 @@
+.about-masthead {
+    font-family: @rway;
+    margin: 3rem 0 1rem 0;
+    line-height: 1.2;
+    font-size: 2.2rem;
+}
+
+.testimonials-board {
+    display: flex;
+    flex-direction: column;
+
+    .testimonial {
+        display: flex;
+        padding: 2rem;
+    
+        .profile-placeholder {
+            border-radius: 50rem;
+            width: 48rem;
+            height: 7rem;
+            background: gray;
+            border: 1px solid black;
+        }
+    
+        .testimonial-text {
+            margin-left: 2rem;
+            font-family: @ssp;
+
+            .name {
+                font-weight: bold;
+                padding: .5rem 0;
+            }
+        }
+    }
+}
+
+
```

Comment:
_In reference to https://github.com/eztip/eztip-UI-joe-pak/blob/613cf4e4d13b2212e23488eed45fc07487008d60/LESS/about.less#L1-L34_

It is apparent you know how to use the powers of LESS.  Well done here, organized, modularized, and excellent use of nesting to achieve that.

Smart use of vars for DRY.